example/spec/helpers/granuleUtils.js

```diff
@@ -144,11 +146,120 @@ async function setupTestGranuleForIngest(bucket, inputPayloadJson, granuleRegex,
   ])(baseInputPayload);
 }
 
-const deleteGranules = async (prefix, granules) => {
+/**
+ * Replicate a granule object from a provided sample to be added to the granules payload of a workflow.
+ * No S3 data staging if the replicated granule being used with the QueueGranulesPassThrough Workflow
+ * @param {boolean} isPassThroughWorkflow - if granuleToReplicate is being used with the QueueGranulesPassthrough workflow logic
+ * @param {Object} granuleToReplicate - granule being replicated
+ * @param {string} bucket - S3 data bucket if NOT using QueueGranulesPassthrough workflow logic
+ * @param {string} granuleRegex - regex to generate the new granule id
+ * @returns {Promise<Object>} - a granule JS object with the updated granule ids
+ */
+async function generatePayloadGranule(isPassThroughWorkflow = true, granuleToReplicate, bucket, granuleRegex) {
+  const oldGranuleId = granuleToReplicate.granuleId;
+  const newGranuleId = randomStringFromRegex(granuleRegex);
+
+  if (isPassThroughWorkflow === true) {
```

Comment:
```suggestion
  if (isPassThroughWorkflow) {
```

Reply:
It's always good to be explicit when checking if a variable is equal to a specific value.